guard-sorbet.gemspec

```
@@ -11,6 +11,6 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency 'guard', '~> 2.0'
   spec.add_runtime_dependency 'guard-compat', '~> 1.0'
-  spec.add_runtime_dependency 'sorbet', '~> 0.5.0'
-  spec.add_runtime_dependency 'sorbet-runtime', '~> 0.5.0'
+  spec.add_runtime_dependency '>= 0.5.0', '< 0.7.0'
```

Missing gem name 'sorbet' as the first argument to add_runtime_dependency. The line should be `spec.add_runtime_dependency 'sorbet', '>= 0.5.0', '< 0.7.0'`.
```suggestion
  spec.add_runtime_dependency 'sorbet', '>= 0.5.0', '< 0.7.0'
```